backend/server/remux/routing_wrappers.go

```diff
@@ -6,10 +6,10 @@ package remux
 import (
 	"net/http"
 
-	"github.com/gorilla/mux"
+	"github.com/go-chi/chi/v5"
 )
 
-// Route rewraps gorilla.mux's Handle/Methods to provide a better at-a-glance reading of route definitions
-func Route(r *mux.Router, method string, path string, handler http.Handler) {
-	r.Handle(path, handler).Methods(method)
+// Route rewraps chi's Handle/Methods to provide a better at-a-glance reading of route definitions
+func Route(r chi.Router, method string, path string, handler http.Handler) {
+	r.Method(method, path, handler)
```

Comment:
:neutral_face: This is basically what the `Route` function does. Probably best to do this the way you did it, but kind of annoying that we're just mirroring their approach.

Reply:
yeah I know right? I initially tried to follow their approach, but imo it made the code slightly less verbose, but it made finding a specific url path more difficult, so I shoehorned their approach into our code